src/debug/daccess/dacdbiimpl.cpp

```diff
@@ -5604,12 +5615,21 @@ void DacDbiInterfaceImpl::GetContext(VMPTR_Thread vmThread, DT_CONTEXT * pContex
     if (pFilterContext == NULL)
     {
         // If the filter context is NULL, then we use the true context of the thread.
+        
+#ifdef FEATURE_PAL        
+        // GetThreadContext() is currently not implemented in PAL because we have no way to convert a thread ID to a 
+        // thread handle.  The function to do the conversion is OpenThread(), which is not implemented in PAL.
+        // Instead, we just zero out the seed CONTEXT for the stackwalk.  This tells the stackwalker to
+        // start the stackwalk with the first explicit frame.  This won't work when we do native debugging, 
+        // but that won't happen on the Linux/Mac since they don't support native debugging.
```

Comment:
Maybe we should just say on PAL instead of Linux/Mac. We will soon have FreeBSD too.